nucypher/cli/actions.py

```diff
@@ -43,45 +43,40 @@ def load_seednodes(min_stake: int,
     return teacher_nodes
 
 
-def destroy_system_configuration(config_class,
-                                 config_file=None,
-                                 network=None,
-                                 config_root=None,
-                                 force=False,
-                                 log=LOG):
+def destroy_configuration_root(config_root=None, force=False, logs: bool = False) -> str:
+    """CAUTION: This will destroy *all* nucypher configuration files from the filesystem"""
 
     config_root = config_root or DEFAULT_CONFIG_ROOT
 
-    try:
-        character_config = config_class.from_configuration_file(filepath=config_file, domains={network})
+    if not force:
+        message = f"Destroy top-level configuration directory: {config_root}?"
+        click.confirm(message, abort=True)  # ABORT
 
-    except FileNotFoundError:
-        config_file_location = config_file or config_class.DEFAULT_CONFIG_FILE_LOCATION
+    shutil.rmtree(config_root, ignore_errors=force)       # config
```

Comment:
```Traceback (most recent call last):
  File "/home/tux/code/nucypher-kms/venv/bin/nucypher", line 11, in <module>
    load_entry_point('nucypher', 'console_scripts', 'nucypher')()
  File "/home/tux/code/nucypher-kms/venv/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/tux/code/nucypher-kms/venv/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/tux/code/nucypher-kms/venv/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/tux/code/nucypher-kms/venv/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/tux/code/nucypher-kms/venv/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/tux/code/nucypher-kms/nucypher/cli/main.py", line 109, in remove
    destroy_configuration_root(config_root=config_root, force=force, logs=logs)
  File "/home/tux/code/nucypher-kms/nucypher/cli/actions.py", line 55, in destroy_configuration_root
    shutil.rmtree(config_root, ignore_errors=force)       # config
  File "/home/tux/code/nucypher-kms/venv/lib/python3.7/shutil.py", line 482, in rmtree
    onerror(os.lstat, path, sys.exc_info())
  File "/home/tux/code/nucypher-kms/venv/lib/python3.7/shutil.py", line 480, in rmtree
    orig_st = os.lstat(path)
FileNotFoundError: [Errno 2] No such file or directory: '/home/tux/.local/share/nucypher'
```

If you try to `remove` when the directory doesn't exist, it throws this error. How about checking if the dir exists before the attempt?

Reply:
:heavy_check_mark: - Pushed the RFC and a test to match.